AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -299,6 +299,16 @@ - (NSUInteger)lineCount
   return lineCount;
 }
 
+- (BOOL)isTruncated
+{
+  if (self.canUseFastPath) {
+    CGRect boundedRect = [_attributes.attributedString boundingRectWithSize:CGSizeMake(_constrainedSize.width, CGFLOAT_MAX) options:NSStringDrawingUsesLineFragmentOrigin | NSStringDrawingTruncatesLastVisibleLine context:nil];
```

Comment:
Could you break this method call across a couple lines because of the long arguments?  Aligning on : should look nice :)